cypress.base.config.ts

```diff
@@ -1,9 +1,9 @@
 import { defineConfig } from 'cypress'
 import fs from 'fs'
-import getCompareSnapshotsPlugin from './src/plugin'
+import getCompareSnapshotsPlugin from './src/plugin.js'
```

Comment:
why this though? 🤔

Reply:
ahh, that is because we are using ts node LTS definition and uses  node16 (tsconfig.json)
It has its meaning in the way that we are used to not including the file termination in our imports but that is wrong because  ES MODULES specification tells you to add the termination. 
SO I think it is a good practice to have the termination set on the imported files , otherways we can avoid to do that but we will need to override our tsconfig.json file and set it back to old node or AMD 
![Uploading image.png…]()
 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
Shouldn't it be ./dest/plugin.js in that case?